Dockerfile.amd64

```
@@ -4,12 +4,31 @@ ARG PROTOC_VERSION=29.3
 ARG CONFIG_FILE=config/config.yaml
 
 ## Rust builder ################################################################
-# Specific debian version so that compatible glibc version is used
-FROM rust:1 AS rust-builder
+FROM ${UBI_MINIMAL_BASE_IMAGE}:${UBI_BASE_IMAGE_TAG} AS rust-builder
 ARG PROTOC_VERSION
 
 ENV CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
 
+# Install dependencies
+RUN microdnf -y update && \
```

nit: disable subscription-manager plugin to avoid log message about not having a subscription.
```suggestion
RUN microdnf --disableplugin=subscription-manager -y update && \
```